Update to use Influx registry and better explanation on how to retrieve property names (#6316)
OpenLiberty/docs: modules/ROOT/pages/micrometer-metrics.adoc

  - For example, if you provide the Influx Micrometer registry library and its dependencies, you must set the `mp.metrics.influx.enabled` property  to `true` to render it active. You can set these properties in your `server.xml` file by using the config:variable[ ] configuration element. For more information about setting MicroProfile Config properties, see xref:external-configuration.adoc[External configuration of microservices].
  - You can also set MicroProfile Config properties for each Micrometer registry to configure the connection to the associated monitoring tool. The properties use the following syntax: `mp.metrics.<registry>.<property>`,  where `<registry>` is the identifying value that is defined for that registry by Micrometer.
  - For example, properties for the Influx registry follow the  `mp.metrics.influx.<property>` syntax. For information about the properties that are available for each registry, see the Micrometer documentation.

> Note that the Micrometer documentation for each Micrometer registry will list, as an example, the properties in a format compliant with Spring Boot's ingestion of these properties. The properties follows the syntax `management.metrics.export.<registry>.<property>`. The `<registry>` and `<property>` values correspond to the `<registry>` and `<property>` values used for the above MicroProfile Config properties. Certain properties that are listed may contain dashes. The underlying Micrometer library reads the properties in camel case format. For example, the Micrometer Influx registry documentation lists an `auto-create-db` property that is consumed by Spring Boot. The Micrometer library accepts it as `autoCreateDb`. The corresponding MicroProfile Config property would be `mp.metrics.influx.autoCreateDb`.

Reviewer: ```suggestion
The Micrometer documentation for each Micrometer registry lists  example properties in a format that is compliant with the Spring Boot ingestion of these properties. Theese example properties follow the `management.metrics.export.<registry>.<property>` syntax. The `<registry>` and `<property>` values correspond to the `<registry>` and `<property>` values that are used for the previously described MicroProfile Config properties. Certain properties that are listed might contain dashes. The underlying Micrometer library reads the properties in camel case format. For example, the Micrometer Influx registry documentation lists an `auto-create-db` property that is consumed by Spring Boot. The Micrometer library accepts it as `autoCreateDb`. The corresponding MicroProfile Config property is `mp.metrics.influx.autoCreateDb`.
```
